[CIAPP-7783] Update PR summaries docs (#19873)
DataDog/documentation: content/en/continuous_integration/guides/pull_request_comments.md

{{< /site-region >}}

If you're using the Test Visibility product, Datadog can integrate with GitHub to show test results summaries directly
in your pull requests. The summary contains an overview of the tests executions, flakiness information, test performance regressions, and

> error messages for failed tests. It also shows how much execution time you have saved thanks to the Intelligent Test Runner.

Reviewer: ```suggestion
error messages for failed tests. When Intelligent Test Runner is enabled, it also shows the test execution time saved.
```

Could we add a link to ITR docs?
